README.md

```diff
@@ -101,6 +102,7 @@ See `dotnet-gitchanges\appsettings.json` for the default settings file. Any sett
 | Repository.OverrideSource | Path to override source (see [Overriding repository changes](#overriding-repository-changes))
 | FileSource | Path to file source (see [Existing repository](#existing-repository)).
 | MultiProject | Specifies whether repository should be processed as a multi project repository.
+| UseGitTags | For version, tags on comments in git will be used.

```

Comment:
"If true, version will be taken from commit tag instead of commit message. All untagged commits will be grouped under the nearest subsequent tagged version."